### ISSUE TITLE
💡[Feature]: Face Recognition using 3 models 

### ISSUE THREAD

**Issue by ShraddhaSabde** [recodehive/machine-learning-repos#961]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Previously I have submitted Face Recognition only using 1 model now I have implemented it in FaceNet, DeepFace, VGGNet 
FaceRecognition library gives us higher Accuracy because no need of any models.
Then Facenet and DeepFace almost gives similar Accuracy.
VGGNet gives low Accuracy.

### Use Case

Numpy
Keras
deepface
PIL
tkinter
cv2
os
tensorflow
mtcnn

### Benefits

-

### Add ScreenShots

-

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
